### ISSUE TITLE
flake: pull-kubernetes-bazel-test //cmd/kubelet/app:go_default_test

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubernetes/kubernetes#81100]
**Which jobs are failing**:
pull-kubernetes-bazel-test

**Which test(s) are failing**:
//cmd/kubelet/app:go_default_test

**Since when has it been failing**:
Flaky since Aug 6, 2019.

**Testgrid link**:
https://k8s-testgrid.appspot.com/presubmits-kubernetes-blocking#pull-kubernetes-bazel-test

PR with varying results: https://github.com/kubernetes/kubernetes/pull/81003
PR with consistent failure: https://github.com/kubernetes/kubernetes/pull/81066
Note that the latter just adds a unit test, should not cause anything else to fail.

**Reason for failure**:
unknown

**Anything else we need to know**:

**Comment by MikeSpreitzer** [comment]
@kubernetes/sig-node-bugs

**Comment by MikeSpreitzer** [comment]
This test finally passed for PR #81066 --- without any change to that PR.

**Comment by liggitt** [comment]
fixed in master in #80236

/close

**Comment by liggitt** [comment]
looks like the feature branch was last rebased on 7/14:

```
$ git merge-base upstream/feature-rate-limiting upstream/master
c6eb9a8ed51f5c63cb351e2a4c13494bf5c303a2

$ git show c6eb9a8ed51f5c63cb351e2a4c13494bf5c303a2
commit c6eb9a8ed51f5c63cb351e2a4c13494bf5c303a2
Merge: 7285829b830 eb1fbc68f6a
Author: Kubernetes Prow Robot <k8s-ci-robot@users.noreply.github.com>
Date:   Sun Jul 14 21:47:05 2019 -0700

    Merge pull request #80151 from nikhita/rules-cleanup
    
    publishing: bump go versions for 1.13 and 1.14
```
